leader-election-impl/src/test/java/com/palantir/paxos/SqlitePaxosStateLogMigrationStateTest.java

```diff
@@ -99,8 +99,8 @@ public void finishingMigrationForOneNamespaceDoesNotSetFlagForOthers() {
     public void finishingMigrationForOneUseCaseDoesNotSetFlagForOthers() {
         migrationState.migrateToValidationState();
 
-        SqlitePaxosStateLogMigrationState otherState = SqlitePaxosStateLogMigrationState
-                .create(ImmutableNamespaceAndUseCase.of(Client.of("namespace"), "other"), connSupplier);
+        SqlitePaxosStateLogMigrationState otherState = new SqlitePaxosStateLogFactory().createMigrationState(
+                ImmutableNamespaceAndUseCase.of(Client.of("namespace"), "other"), connSupplier);
```

Comment:
These don't happen simultaneously with the other instances in the other tests in this test class, so while this _might_ be a bit sketchy it's ok